bot/telegram/send_image.go

```diff
@@ -0,0 +1,95 @@
+package telegram
+
+import (
+	"context"
+	"fmt"
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"jsin/bot/message_handler"
+	"jsin/logger"
+	"time"
+)
+
+func (b *Bot) SendImage(update tgbotapi.Update, object message_handler.ObjectDTO) error {
+	file := tgbotapi.FileBytes{
+		Name:  object.ObjectKey,
+		Bytes: object.Object,
+	}
+	message, err := b.bot.Send(
+		tgbotapi.PhotoConfig{
+			BaseFile: tgbotapi.BaseFile{
+				BaseChat: tgbotapi.BaseChat{
+					ChatID:           update.Message.Chat.ID,
+					ReplyToMessageID: 0,
+				},
+				File: file,
+			},
+		},
+	)
+	if err != nil {
+		logger.Errorf("===== Send image failed: %+v", err.Error())
+		return err
+	}
+	logger.Infof("Image sent, message id: %d", message.MessageID)
+
+	return nil
+}
+
+func (b *Bot) SendImageByObject(chatID int64, object message_handler.ObjectDTO) error {
+	file := tgbotapi.FileBytes{
+		Name:  object.ObjectKey,
+		Bytes: object.Object,
+	}
+
+	message, err := b.bot.Send(
+		tgbotapi.PhotoConfig{
+			BaseFile: tgbotapi.BaseFile{
+				BaseChat: tgbotapi.BaseChat{
+					ChatID: chatID,
+				},
+				File: file,
+			},
+			Caption: "🌅 Have a great day!",
```

Comment:
Please remember that we put these strings like magic key words or numbers in a constant value!